docs/user-guide/install-zowe-pswi-deployment.md

```diff
@@ -41,26 +48,72 @@ Before installing, ensure the [z/OSMF requirements](install-zowe-pswi-address-re
-    :::
-    
     8. Complete **Mount Workflow** to mount the Zowe zFS. Complete both steps in the workflow. Perform the following steps to execute each step individually:
        1. Click the title of the step.
-       2. Select the Perform tab.
+       2. Select the **Perform** tab.
        3. Review the step contents and update the JCL if needed.
-       4. Select Next.
-       5. Repeat the previous two steps to complete all items until the Finish option is available. 
+       4. Select **Next**.
+       5. Repeat the previous two steps to complete all items until the **Finish** option is available.
 
     9. (Optional) To configure your Zowe instance, follow the procedure in [Configuring Zowe with z/OSMF Workflows](configure-zowe-zosmf-workflow.md).
 
    10. Specify the name and description of the new target software instance.
 
-        - All workflows that are mentioned in the previous steps are part of the PSWI.  
-        **Note:** You do not have to execute all workflows during PSWI provisioning in z/OSMF immediately.
+:::note Notes
+* All workflows that are mentioned in the previous steps are part of the PSWI.  
+* You do not have to execute all workflows during PSWI provisioning in z/OSMF immediately.
+:::
 
 The deployment process is complete. The new software instance is defined to z/OSMF. You are now ready to Import Product Information into z/OSMF before you install product maintenance.
+
+## Recommendations
+
+### Cleanup
+
+If there is a need to update the deployment settings such as the HLQ after a previour run, consider that the deployment process does not perform a full cleanup of the datasets.
+Before attempting deployment again, make sure the following datasets are deleted to avoid write conflicts:
+
+```plaintext
+{TARGET_HLQ}.PRODUCT.ZOWE.V2.CSI
+{TARGET_HLQ}.PRODUCT.ZOWE.V2.D.AZWEAUTH.#
+{TARGET_HLQ}.PRODUCT.ZOWE.V2.D.AZWESAMP.#
+{TARGET_HLQ}.PRODUCT.ZOWE.V2.D.AZWEZFS.#
+{TARGET_HLQ}.PRODUCT.ZOWE.V2.SMPLOG.#
+{TARGET_HLQ}.PRODUCT.ZOWE.V2.SMPLOGA.#
+{TARGET_HLQ}.PRODUCT.ZOWE.V2.SMPLTS.#
+{TARGET_HLQ}.PRODUCT.ZOWE.V2.SMPMTS.#
+{TARGET_HLQ}.PRODUCT.ZOWE.V2.SMPPTS.#
+{TARGET_HLQ}.PRODUCT.ZOWE.V2.SMPSCDS.#
+{TARGET_HLQ}.PRODUCT.ZOWE.V2.SMPSTS.#
+{TARGET_HLQ}.PRODUCT.ZOWE.V2.T.SZWEAUTH.#
+{TARGET_HLQ}.PRODUCT.ZOWE.V2.T.SZWEEXEC.#
+{TARGET_HLQ}.PRODUCT.ZOWE.V2.T.SZWELOAD.#
+{TARGET_HLQ}.PRODUCT.ZOWE.V2.T.SZWESAMP.#
+{TARGET_HLQ}.PRODUCT.ZOWE.V2.WORKFLOW.#
+{TARGET_HLQ}.PRODUCT.ZOWE.V2.ZFS.#
+```
+
+If deployment options must be modified and the JCL recreated, first check the original JCL for the temporary ZFS dataset name.
+
+**Example:**
+`<user>.SWDEPL.<random_id>.ZFS`
+ 
+ Delete this name manually. 
+ 
+ :::note 
+ Note that the deployment job is unable to clean this data set name once this is recreated because the random id will change.
```

Comment:
What is "this" in the sentence "once this is recreated..."? Please replace "this" with the specific thing that is being recreated.

Reply:
"this" refers to the dataset a few lines above - <user>.SWDEPL.<random_id>.ZFS
The note should clarify why the manual cleanup is needed.

Reply:
Is this rewritten note accurate:
Note that the deployment job is unable to clean this data set name once the data set is recreated because the value for `random_id` will change.

Reply:
Should this note also be included in v2.18? Currently it is not.

Reply:
looks better. I synced the changes with v2.18